### PR TITLE
docs(auth): improve auth with service account doc

### PR DESCRIPTION
**Pull request by aureldent**
Deployed doc => https://lumapps.github.io/lumapps-sdk/authentication/

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -34,7 +34,52 @@ make check
...
+
+* feat: A new feature
+* fix: A bug fix
+* docs: Documentation only changes
+* lint: Changes that do not affect the meaning of the code (white-space, formatting, missing semicolons, etc)
+* refactor: A code change that neither fixes a bug or adds a feature
+* test: Adding missing tests or correcting existing tests
+* chore: Changes to the build process or auxiliary tools and libraries such as distribution generation
+
+Example:
```
Comment (tribouille) [inline docs/CONTRIBUTING.md:70]: I would have put the example before the Type

File: docs/authentication.md [hunk 2]
```diff
@@ -32,7 +32,86 @@ client = BaseClient(token="<your_token>")
...
+        With your service account get a google access token
+        <div style="margin: 8px;">
+        To know how to get this token with curl and you service account follow <a href="https://gist.github.com/ryu1kn/c76aed0af8728f659730d9c26c9ee0ed"> this tutorial</a>
+        <br/>
+        For an extended documentation on that process you can follow the <a href="https://developers.google.com/identity/protocols/oauth2/service-account"> google documentation</a>
+        </div>
+    </li>
+    <li>
+        Use this token as the bearer token to call the <a href="https://apiv1.lumapps.com/#operation/User/Gettoken"> user/get endpoint</a>
+        <br/>
```
Comment (tribouille) [inline docs/authentication.md:53]: Does not render correctly

File: docs/authentication.md [hunk 3]
```diff
@@ -42,15 +121,15 @@ my_platform_id="<your_plaform_id>"
 user_to_authenticate_on_behalf_of = "<user_email>"
 
 client = BaseClient(
-    token="<your_token>",
     auth_info=my_sa)
```
Comment (tribouille) [inline docs/authentication.md:124]: my_sa is not clear what it means... (not in this PR)

File: docs/authentication.md (hunk 2 above)
Comment (aureldent) [reply docs/authentication.md:53]: what do you mean by does not render correctly ? the link works so ...

Comment (tribouille) [reply docs/authentication.md:53]: image

Comment (aureldent) [reply docs/authentication.md:53]: It renders correctly on my side when I go to the page, on firefox or chrome ....

**Review by tribouille — APPROVED** [review]

**Pull request merged**